src/libgap-api.h

```diff
@@ -4,6 +4,7 @@
 #define LIBGAP_API_H
 
 #include "gap.h"
+#include "plist.h"
```

Comment:
Why include this here? It should be included in the C file only, I'd say, precisely to avoid exposing these internals to users of libgap, no?

Reply:
Oh yes, this is simply the wrong file. Thank you :)